language_clean_patch.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+from unidiff import PatchSet
+from unidiff.constants import LINE_TYPE_EMPTY
+
+import argparse
+import os
+
+SUPPORTED_LANGUAGES = ["ar-EG", "ca-ES", "cs-CZ", "da-DK", "de-DE", "en-GB", "en-US", "es-ES",\
+                       "fi-FI", "fr-FR", "hu-HU", "it-IT", "ja-JP", "ko-KR", "nb-NO", "nl-NL",\
+                       "pl-PL", "pt-BR", "ru-RU", "sv-SE", "tr-TR", "zh-CN", "zh-TW"]
```

Comment:
In the future we should probably have all scripts source the languages from a single place

Reply:
Yes, definitely. I was also thinking we should probably move the language*.py tools into a Python package of some sort. Another item for the to do list…